sdk/monitor/monitor-query/samples-dev/metricsQuery.ts

```diff
@@ -24,26 +24,36 @@ export async function main() {
   const iterator = metricsQueryClient.listMetricDefinitions(metricsResourceId);
   let result = await iterator.next();
   const firstMetric: MetricDefinition = result.value;
-
-  while (!result.done) {
-    console.log(` metricDefinitions - ${result.value.id}, ${result.value.name}`);
-    result = await iterator.next();
+  let secondMetricName: string = "TotalCalls";
```

Comment:
What's the significance of "TotalCalls"? Also what's the reason for not collecting all the metric names and passing them to `query`? It feels really strange to iterate over each result and assign the last one to something called `secondMetricName`.

Reply:
Nope I was just testing something there. Removed that and changed sample to pass in all metric names